mongodbatlas/private_endpoints.go

```diff
@@ -51,35 +51,49 @@ var _ PrivateEndpointsService = &PrivateEndpointsServiceOp{}
 // PrivateEndpointConnection represents MongoDB Private Endpoint Connection.
 type PrivateEndpointConnection struct {
 	ID                           string   `json:"id,omitempty"`                           // Unique identifier of the AWS PrivateLink connection or Azure Private Link Service.
-	ProviderName                 string   `json:"providerName,omitempty"`                 // Name of the cloud provider for which you want to create the private endpoint service. Atlas accepts AWS or AZURE.
+	ProviderName                 string   `json:"providerName,omitempty"`                 // Name of the cloud provider for which you want to create the private endpoint service. Atlas accepts AWS, AZURE or GCP.
 	Region                       string   `json:"region,omitempty"`                       // Cloud provider region for which you want to create the private endpoint service.
 	EndpointServiceName          string   `json:"endpointServiceName,omitempty"`          // Name of the PrivateLink endpoint service in AWS. Returns null while the endpoint service is being created.
-	ErrorMessage                 string   `json:"errorMessage,omitempty"`                 // Error message pertaining to the AWS PrivateLink connection or Azure Private Link Service. Returns null if there are no errors.
+	ErrorMessage                 string   `json:"errorMessage,omitempty"`                 // Error message pertaining to the AWS PrivateLink connection or Azure Private Link Service or GCP. Returns null if there are no errors.
```

Comment:
[nit] Since there is service name in error message description, I think we should put `GCP Private Service Connect` instead of `GCP`. Also, there is a missing comma (punctuation) after `AWS PrivateLink connection, `.